### PR TITLE
Fallback datepicker if needed

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fallback datepicker if needed
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/FF-91?filter=-1
| How to test?  | install langage and see your datepiacker

ping @AlexEven 

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: classes/Media.php [hunk 1]
```diff
@@ -66,6 +66,34 @@ class MediaCore
         'effects.transfer' => array('fileName' => 'jquery.effects.transfer.min.js', 'dependencies' => array('effects.core'), 'theme' => false)
     );
 
+    private static $jquery_ui_datepicker_locale = array(
+        'bg' => 'bg',
```
Comment (Quetzacoalt91) [inline classes/Media.php]: Do we really need to add a line for a fallback on the same language ?

Comment (aleeks) [inline classes/Media.php]: Loul.. libre office owned me ..

Comment (aleeks) [inline classes/Media.php]: Done.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by AlexEven** [comment]
Tested with Mexican and Vietnamese, date picker now translated.
We're good to go :)
Thanks @aleeks!

**Comment by Quetzacoalt91** [comment]
Thanks Alex**s**

**Pull request merged**